README.md

```diff
@@ -38,765 +56,196 @@ Given that Webpacker already provides React integration, why would you use "Reac
-+ Currently, the only option you may pass is `replay_console` (boolean)
-
-This is a helper method that takes any JavaScript expression and returns the output from evaluating it. If you have more than one line that needs to be executed, wrap it in an IIFE. JS exceptions will be caught and console messages handled properly.
+[React Router](https://github.com/reactjs/react-router) is supported, including server-side rendering! See:
 
-## Multiple React Components on a Page with One Store
-You may wish to have 2 React components share the same the Redux store. For example, if your navbar is a React component, you may want it to use the same store as your component in the main area of the page. You may even want multiple React components in the main area, which allows for greater modularity. Also, you may want this to work with Turbolinks to minimize reloading the JavaScript. A good example of this would be something like a notifications counter in a header. As each notification is read in the body of the page, you would like to update the header. If both the header and body share the same Redux store, then this is trivial. Otherwise, we have to rely on other solutions, such as the header polling the server to see how many unread notifications exist.
+1. [React on Rails docs for react-router](docs/additional-reading/react-router.md)
+2. Examples in [spec/dummy/app/views/react_router](./spec/dummy/app/views/react_router) and follow to the JavaScript code in the [spec/dummy/client/app/startup/ServerRouterApp.jsx](spec/dummy/client/app/startup/ServerRouterApp.jsx).
+3. [Code Splitting docs](docs/additional-reading/code-splitting.md) for information about how to set up code splitting for server rendered routes.
 
-Suppose the Redux store is called `appStore`, and you have 3 React components that each needs to connect to a store: `NavbarApp`, `CommentsApp`, and `BlogsApp`. I named them with `App` to indicate that they are the registered components.
+# Caching and Performance
+Caching and performance optimizations are now part of [React on Rails Pro](https://github.com/shakacode/react_on_rails/wiki).
 
-You will need to make a function that can create the store you will be using for all components and register it via the `registerStore` method. Note: this is a **storeCreator**, meaning that it is a function that takes (props, location) and returns a store:
+### Deployment
 
-```js
-function appStore(props, railsContext) {
-  // Create a hydrated redux store, using props and the railsContext (object with
-  // Rails contextual information).
-  return myAppStore;
-}
+Check out our [Deployment Guide](docs/basics/deployment.md).
 
-ReactOnRails.registerStore({
-  appStore
-});
-```
+## Error Handling
```

Comment:
We should add a [blank line before the header](https://github.com/markdownlint/markdownlint/blob/e3dbc768e1608da5baab0a5896c54e840ff59110/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines).